tests/cypress/e2e/contentEditor/boundComponent.cy.ts

```diff
@@ -0,0 +1,42 @@
+import {JContent} from '../../page-object/jcontent';
+import {enableModule} from '@jahia/cypress';
+import {PickerField} from '../../page-object/fields';
```

Comment:
to be removed

Reply:
There was a lint warning: `PickerField is defined but never used`. not urgent, but I was adding this since we might need some additonal changes as well.